keyboards/pabile/p40/keymaps/ortho/config.h

```diff
@@ -0,0 +1,18 @@
+
+#include "../../config.h"
+
+/* pin-out for PCB v2 */
+#undef MATRIX_ROW_PINS
+#define MATRIX_ROW_PINS { D1, D0, F4, F5 }
+
+#undef MATRIX_COL_PINS
+#define MATRIX_COL_PINS { D4, C6, D7, E6, B4, B2, B3, B1, F7, F6 }

```

Comment:
This should be a hardware revision, rather than a separate keymap.

You can look at the planck for reference on how to implement this.